src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,47 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+public class Book {
+
+    @Id
+    private long id;
+
+    private String genre;
+
+    @Column(nullable = false)
```

Comment:
Try the annotation `@NotNull` (javax.validation.constraints.NotNull) instead of `@Column(nullable=false)`. The first one checks for null before the persistance layer :) (fail fast principle)

Reply:
Solved.